src/olympia/lib/settings_base.py

```diff
@@ -1616,7 +1617,7 @@ def get_redis_settings(uri):
 
 # If you want to allow self-reviews for add-ons/apps, then enable this.
 # In production we do not want to allow this.
-ALLOW_SELF_REVIEWS = False
+ALLOW_SELF_REVIEWS = True
```

Comment:
I wasn't 100% about this change, but _only_ prod sets it to `False` so it seemed more efficient to default it to `True` in one file than set it to `False` in three.

Reply:
Hmm, this is one of the things where I'm unsure as well and would tend to default to the safest value and overwrite it specifically for dev and stage to make things less "safe".